site/docs/v1/tech/admin-guide/proxy-setup.adoc

```diff
@@ -100,6 +100,11 @@ Never cache FusionAuth non-static asset responses.
 
 The community has provided a number of example configurations for different proxies. You can https://github.com/FusionAuth/fusionauth-contrib/tree/master/Reverse%20Proxy%20Configurations[view them in the GitHub repo].
 
+[NOTE.note]
+====
+Make sure your proxy is using at least HTTP/1.1, as we do not support HTTP/1.0 anymore.
```

Comment:
Can you please note when 1.0 support was dropped? https://fusionauth.io/docs/v1/tech/release-notes should tell you.